src/sync/impl/sync_file.hpp

```diff
@@ -59,19 +61,21 @@ void remove_nonempty_dir(const std::string& path);
 
 class SyncFileManager {
 public:
+    using StringPair = std::tuple<std::string, std::string>;
```

Comment:
No, you're right: a struct makes more sense. I'm still stuck in a Swift mindset.